README.md

```diff
@@ -68,7 +73,7 @@ docker exec -it kind-control-plane crictl images
 ```
 
 ### Deploying in a Cluster
-1. Create chart values file `yourvaluesfile.yaml`. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) on how to adjust this and update the `kubeslice-controller` image to the local build image.
+1. Create the chart values `yourvaluesfile.yaml` file. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) on how to adjust this and update the `kubeslice-controller` image to the local build image.
```

Comment:
The previous usage was correct - 

Add a comma after file 

Rephrase it as `Create a chart values file called yourvaluesfile.yaml. `

Reply:
updated